site/_posts/2022-06-28-remix-demo.md

```diff
@@ -0,0 +1,55 @@
+
+* Offer external authentication by trusted sources such as Google, Facebook, or Apple (known as Identity Providers)
+* Set up your org as an Identity Provider for other apps
+* Access external APIs that require specific kinds of authentication
+* Protect your own APIs from unauthorized access
+* Limit the scope of authentication to a specific domain or client
+* Let you sleep easily at night knowing that your solution has been professionally security audited for years
+
+But to do so, you need to adopt modern standards and practices -- and the sooner the better. In fact adopting a standards-based third-party server will result in less code than a hand-rolled solution.
+
+## Setting up FusionAuth with Remix
+
+We've prepared a small package with just the parts of Remix that are necessary to show how to log in and out, and require authentication for a given component. Clone [this repository](https://github.com/FusionAuth/fusionauth-example-remix) and install dependencies, and be sure to check the README.
+
+There's one somewhat unusual feature of this example, which is that we chose file-based rather than cookie-based session storage. One of the principles of authentication is that you should avoid moving sensitive data around to and from the user-facing client whenever possible, which is why we always recommend a server component for every front-end technology. For instance our [React demo](https://github.com/FusionAuth/fusionauth-example-react-2.0) has a client written in React that talks to an Express server which talks to the FusionAuth server. If you compare Remix to React, one of the biggest differences is that Remix has the server built in, so there's no need to run a separate server process.
+
+File-based session storage has one drawback that is only relevant at scale: once you need more than one Remix server, you will have to pin each user to a particular server for the duration of that session. However most developers are unlikely to encounter that kind of problem -- or by the time you do, you'll have the resources to solve it.
+
+You will also need a working FusionAuth instance. The easiest way to get this for testing purposes is to have a short conversation with a member of our [Sales team](https://fusionauth.io/contact) who will be able to provision a cloud instance for you. The second fastest way is to use the Docker install. The third fastest way is to use the Fast Path install detailed in our [Five Minute Setup Guide](https://fusionauth.io/docs/v1/tech/5-minute-setup-guide). In all cases you should install the development data included in our [Kickstart project](https://github.com/FusionAuth/fusionauth-example-kickstart/tree/master/fusionauth).
+
+Once you have a FusionAuth instance, you should be able to go to the admin page and configure your FusionAuth instance like this:
+
+<img src="/assets/img/blogs/connecting-fusionauth-remix/fusionauth-config.png" />
```

Comment:
Please use the image.liquid include. See an example here: https://raw.githubusercontent.com/FusionAuth/fusionauth-site/master/site/_posts/2022-04-25-single-sign-on-with-discord.md

`{% include _image.liquid src="/assets/img/blogs/single-sign-on-discord/discord-application.png" alt="Discord Application." class="img-fluid" figure=true %}`